easybuild/easyconfigs/p/PLINK/PLINK-2.00a3.7-foss-2022a.eb

```
@@ -0,0 +1,58 @@
+easyblock = 'Bundle'
+
+name = 'PLINK'
+version = '2.00a3.7'
+
+homepage = 'https://www.cog-genomics.org/plink/2.0/'
+description = """PLINK is a free, open-source whole genome association analysis toolset,
+designed to perform a range of basic, large-scale analyses in a computationally
+efficient manner."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
```

pretty sure this isn't MPI-aware, so could drop it to `gfbf`
```suggestion
toolchain = {'name': 'gfbf', 'version': '2022a'}
```

---

indeed, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/16940/commits/306631dcc1f95b76bf001dca49b40fc806e2d491

---

it was decided in Slack that we should stick to `foss` for `2022a`, to avoid a mixed situation.

---

@smoors thanks, change reverted in https://github.com/easybuilders/easybuild-easyconfigs/pull/16940/commits/f723e1f6f06bc365061d7a84f042dee6eda5e156